### ISSUE TITLE
Padding stops working for ":" character

### ISSUE THREAD

**Issue by epiales** [adrianlee44/atom-aligner#40]
I have both `Left space for :` and `Right space for :` checked and the `Padding for :` set to `left`

The only workaround currently is going into settings and re-toggling the `Left space for :` option. After awhile, the setting will be ignored and I am forced to do the workaround again.

Expected behavior:
```js
var expected = {
  'one'   : 1,
  'five'  : 5,
  'seven' : 7
}
```

Actual behavior:
```js
var actual = {
  'one': 1,
  'five': 5,
  'seven': 7
}
```

**Comment by openam** [comment]
I had this same issue, and the workaround worked for me.

**Comment by Legioneer** [comment]
:+1:

**Comment by adrianlee44** [comment]
Thanks for reporting this. This should be fixed with v0.17.1.
